service/src/main/java/uk/gov/hmcts/reform/fpl/model/judicialmessage/JudicialMessageMetaData.java

```diff
@@ -12,6 +13,7 @@
 public class JudicialMessageMetaData {
     private final String sender;
     private final String recipient;
-    private final String requestedBy;
+    @JsonProperty("requestedBy")
+    private final String subject;
```

Comment:
would be nice to rename "requestedBy" as subject in the ccd definition

Reply:
it would require data migration